test/torchaudio_unittest/prototype/hdemucs_cpu_test.py

```diff
@@ -1,8 +1,8 @@
 import torch
 from torchaudio_unittest.common_utils import PytorchTestCase
-from torchaudio_unittest.prototype.hdemucs_test_impl import HDemucsTests
+from torchaudio_unittest.prototype.hdemucs_test_impl import HDemucsTests, TestDemucsIntegration
```

Comment:
we have integration test in `test/integration_test` directory, so can we call this something else?